typescript/ccip-server/src/services/__mocks__/HyperlaneService.ts

```diff
@@ -1,7 +1,7 @@
-import { MessageTx } from 'hyperlane-explorer/src/types';
+import { MessageTx } from '../explorerTypes';
 
 class HyperlaneService {
-  async getOriginBlockByMessageId(id: string): Promise<MessageTx> {
+  async getOriginBlockByMessageId(_id: string): Promise<MessageTx> {
```

Comment:
assuming `id` is a field on `MessageTx`, can we make this `messageId: string` instead of `_id`?

Reply:
nit^